docs/going-into-production.rst

```diff
@@ -0,0 +1,353 @@
+.. _going-into-production:
+
+=====================
+Going into production
+=====================
+
+Running CrateDB in different environments requires different approaches. This
+document outlines the basics you need to consider when going into production.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _prod-bootstrapping:
+
+Configure bootstrapping
+=======================
+
+The process of forming a cluster is known as *bootstrapping*. Consult the
+how-to guide on :ref:`CrateDB multi-node setups <multi_node_setup>` for an
+overview of the two different ways to bootstrap a cluster.
+
+If you have been using CrateDB for development on your local machine, there is
+a good chance you have been using :ref:`single host auto-bootstrapping
+<auto-bootstrapping>`.
+
+For improved performance and resiliency, production CrateDB clusters should be
+run with one node per host machine. To do this, you must manually configure the
+bootstrapping process by telling nodes how to:
+
+  a. :ref:`Discover other nodes <discovery>`, and
+  b. :ref:`Elect a master node <master-node-election>`
+
+This is known as *manual bootstrapping*. See the :ref:`how-to guide
+<manual-bootstrapping>` for more information about how bootstrap a cluster
+manually.
+
+Switching to a manual bootstrapping configuration is the first step towards
+going into production.
+
+
+.. _prod-naming:
+
+Naming
+======
+
+
+.. _prod-cluster-name:
+
+Configure a logical cluster name
+--------------------------------
+
+The `cluster.name`_ setting allows you to override the default cluster name of
+``crate``. You should use this setting to give a logical name to your cluster.
+
+For example, add this to your `configuration`_ file:
+
+.. code-block:: yaml
+
+    cluster.name: acme-prod
+
+The ``acme-prod`` name suggests that this cluster is the production cluster for
+the *Acme* organization. If *Acme* has a cluster running in a staging
+environment, you might want to name it ``acme-staging``. This way, you can
+differentiate your clusters by name (visible in the `Admin UI`_).
+
+.. TIP::
+
+    A node will refuse to join a cluster if the respective cluster names
+    do not match
+
+.. SEEALSO::
+
+    :ref:`Cluster names for multi-node setups <multi-node-cluster-name>`
+
+
+.. _prod-node-names:
+
+Configure logical node names
+----------------------------
+
+By default, CrateDB sets the node name for you. You can override this by
+configuring a explicit node name using the `node.name`_ setting in your
+`configuration`_ file.
+
+Automatic nodes names are chosen the `sys.summits`_ table. This is useful in
+development when you don't care too much about cluster architecture. However,
```

Comment:
"when you don't care too much about cluster architecture" --> phrasing sounds a bit weird?